docs/developer-guide/local-development-gotchas.md

```diff
@@ -1,7 +1,32 @@
 # Local development gotchas
-
 This page lists common local development issues and how do resolve them.
 
-## Brief description of problem
 
-Solution description goes here.
+Sometimes you need to disable Django's DEBUG mode in order to test certain behaviours. To do this, you have two options:
+
+1. Add `DEBUG = False` to `config/settings/local.py` 
+2. Add `DEBUG=False` to your `.env` file
```

Comment:
Shouldn't this content be under one of the headings below?

Reply:
Done